test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -541,27 +576,12 @@ define([
             {
                 type: jcm.BACKEND_RESPONSES.LOGS,
                 message: {
-                    job_id: TEST_JOB_ID,
-                    latest: true,
-                    logs: [
-                        {
-                            line: 'this',
-                        },
-                    ],
+                    someJob: JobsData.example.Logs.valid[0],
```

Comment:
logs data from the backend is returned as an object with job ID as key and log data as value